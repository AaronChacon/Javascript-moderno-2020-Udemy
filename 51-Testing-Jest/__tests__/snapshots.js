const cliente = { 
    nombre: 'Aaron Chacon',
    balane: 500,
    tipo: 'Premium'
};




describe('Testing al cliente', () => {
    test('Es Aaron Chacon', () => {
        expect(cliente).toMatchSnapshot();
    });

    
});