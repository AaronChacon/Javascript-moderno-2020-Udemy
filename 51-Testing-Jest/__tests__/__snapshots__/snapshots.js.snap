// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Aaron Chacon 1`] = `
Object {
  "balane": 500,
  "nombre": "Aaron Chacon",
  "tipo": "Premium",
}
`;
